index.html

```diff
@@ -329,7 +329,7 @@ <h2 class="form-h2">I'm always<br>interested in hearing<br>about new projects,<b
                     <h3 class="form-h3">I'm always interested<br>in hearing about new<br>projects, so if you'd<br>like

```

Comment:
Please add unnecessary files and folders like `.vscode` to .gitignore.

Reply:
## Thank you for your feedback @SajjadAhmad14.
The changes have been implemented.